workspace.jsonc

```diff
@@ -251,6 +251,7 @@
         "mousetrap": "1.6.5",
         "nanoid": "3.1.20",
         "nerf-dart": "1.0.0",
+        "new-url-loader": "0.1.1",
```

Comment:
this should be added to the react component.json

Reply:
ok

Reply:
moved